miner_config/README.md

```diff
@@ -0,0 +1,21 @@
+#
+#### Set up helium-assets buckets
+```
+$ gsutil mb \
+    -p nebra-production \
+    -c standard \
+    -l us \
+    gs://{helium-assets,helium-assets-stage}.nebracdn.com
+
+$ gsutil iam ch allUsers:objectViewer \
+    gs://{helium-assets,helium-assets-stage}.nebracdn.com
```

Comment:
this should be helium-snapshots and helium-snapshots-stage no?

Reply:
and a couple more instances in the same file

Reply:
i disagree, because then we have to push an update down to the miner container and to all miners, and also means that if we have issues with nebracdn.com on cloudflare the docker.config cant download.

think it is better over all to leave helium-assets on nebra.com